ArduCopter/AP_Arming_Copter.cpp

```diff
@@ -566,6 +566,16 @@ bool AP_Arming_Copter::alt_checks(bool display_failure)
 //  has side-effect that logging is started
 bool AP_Arming_Copter::arm_checks(AP_Arming::Method method)
 {
+    if (method == AP_Arming::Method::RUDDER) {
```

Comment:
as discussed, it should be safe to remove this bit of code because it shouldn't be a problem if users use the regular rudder arming gesture while in toymode